### PR TITLE
Fork out NotAllowedToFail

### PR DESCRIPTION
**Pull request by bvbfan**
Correct PR for `NotAllowedToFail` plan for 1.5.x hardfork.

### CODE REVIEW

**Comment by bvbfan** [comment]
It's 1.5.x now.
Fixes https://github.com/cakedefi/defichain-private/issues/440

File: src/chainparams.cpp [hunk 1]
```diff
@@ -151,10 +152,13 @@ class CMainParams : public CChainParams {
...
         consensus.mn.creationFee = 10 * COIN;
         consensus.mn.collateralAmount = 1000000 * COIN;
+        consensus.mn.collateralAmountDakota = 20000 * COIN;
         consensus.mn.historyFrame = 300;
         consensus.mn.anchoringTeamSize = 5;
         consensus.mn.anchoringFrequency = 15;
-        consensus.mn.anchoringLag = 15;
+
+        consensus.mn.anchoringTimeDepth = 3 * 60 * 60; // 3 hours
+        consensus.mn.anchoringTeamChange = 120; // Number of blocks
```
Comment (uzyn) [inline src/chainparams.cpp:161]: Is this intended to be in this PR?

**Review by uzyn — COMMENTED** [review]
Is this a PR on "NotAllowedToFail" fix or on Dakota preparation? There seems to be quite some code in there that are not related to "NotAllowedToFail".

**Comment by monstrobishi** [comment]
@uzyn because the base is currently set to master, it should be 1.5.x

**Review by monstrobishi — APPROVED** [review]

**Pull request merged**